.gitignore

```diff
@@ -17,4 +17,6 @@
 /tmp
 
 # Ignore Heroku app binstubs
-/bin/*
\ No newline at end of file
+/bin/*
+
+/Gemfile.lock
```

Comment:
I never do that 😝
It means things like test frameworks and other development gems won't be locked